nss-cohort-37/Workforce-Blue-Barracudas
# MG Department Views

# Description
Created list, details, and create functionality for Departments!
Fixes #5, #6, #7 

## Type of change

# Testing Instructions
- Click on the Departments section! This should take you to a list with all department names, budgets, and the amount of people currently staffed in the department.
-When you click on the Details for a specific department, it should show you the name, budget, and the full names of the employees in the department!
-Lastly, you should be able to create a new department by entering its name and budget!

# Checklist:
